src/applications/gi/tests/00-required.e2e.spec.js

```diff
@@ -1,37 +1,65 @@
 const E2eHelpers = require('../../../platform/testing/e2e/helpers');
 const Timeouts = require('../../../platform/testing/e2e/timeouts');
 const GiHelpers = require('./gibct-helpers');
+const institutionProfile = require('./e2e/institution-profile.json');
 
 module.exports = E2eHelpers.createE2eTest(client => {
+  const institutionAttributes = institutionProfile.data.attributes;
+
   GiHelpers.initApplicationMock();
 
   client.openUrl(`${E2eHelpers.baseUrl}/gi-bill-comparison-tool/`);
 
   E2eHelpers.overrideSmoothScrolling(client);
   client.timeoutsAsyncScript(2000);
 
+  // Landing Page
   client
     .waitForElementVisible('body', Timeouts.normal)
     .waitForElementVisible('.gi-app', Timeouts.verySlow)
     .axeCheck('.main');
+  GiHelpers.searchForInstitution(client);
 
-  client
-    .waitForElementVisible(
-      '.keyword-search input[type="text"]',
-      Timeouts.normal,
-    )
-    .clearValue('.keyword-search input[type="text"]')
-    .setValue('.keyword-search input[type="text"]', 'washington dc');
+  // Search Page
+  GiHelpers.expectLocation(
+    client,
+    `/search?category=school&name=${institutionAttributes.name.replace(
+      /\s/g,
+      '+',
+    )}`,
+  );
+  GiHelpers.selectFirstSearchResult(client);
 
+  // Profile Page
+  GiHelpers.expectLocation(
+    client,
+    `/profile/${institutionAttributes.facility_code}`,
+  );
   client
-    .click('#search-button')
-    .waitForElementVisible('.search-page', Timeouts.normal)
-    .axeCheck('.main');
-  client
-    .waitForElementVisible('.search-result a', Timeouts.normal)
-    .click('.search-result a')
     .waitForElementVisible('.profile-page', Timeouts.normal)
-    .axeCheck('.main');
+    .axeCheck('.main'); // commented out until 7727 is fixed
```

Comment:
When you're ready to move this PR out of draft, let's move this `axeCheck` to after the accordions are opened. We're looking to increase test coverage, and axe doesn't check things that are hidden using `display: none` or other CSS means.

Reply:
By default all the major section accordions are open when loading a profile page
See https://staging.va.gov/gi-bill-comparison-tool/profile/14900416

The "Eligibility Details" section is expanded in `GiHelpers.editEligibilityDetails(client);` and an axe check is performed on that section specifically after it is expanded.